spec/models/feed_spec.rb

```diff
@@ -29,10 +29,6 @@
   end
 
   it "is not expired when new" do
-    expect(feed.created_at).to be_nil
-    expect(feed.fetched_at).to be_nil
-    expect(feed.expired?).to be_falsey
```

Comment:
is `created_at` set in `new` now? I had this all in because I crashed myself by checking `expired?` before `save`

Reply:
The timestamp fields (created_at/updated_at) are only set the first time the object is saved. Line 35 calls `expired?` before we save, so this still checks that.

Reply:
but now we don't have a test for when `fetched_at` is nil and `created_at` is set?

Reply:
ugh apparently I did this twice and this was the bad version. oops.

Reply:
took it out!